modules/extensions-spring-support/src/main/resources/META-INF/mule-module.properties

```diff
@@ -4,14 +4,20 @@ artifact.export.classPackages=org.mule.runtime.module.extension.internal.capabil
                               org.mule.runtime.module.extension.internal.capability.xml.schema.builder,\
                               org.mule.runtime.module.extension.internal.capability.xml.schema.model,\
                               org.mule.runtime.module.extension.internal.config,\
+                              org.mule.runtime.module.extension.internal.runtime.execution,\
+                              org.mule.runtime.module.extension.internal.runtime.operation,\
                               org.mule.runtime.module.extension.internal.resources,\
                               javax.annotation.processing,\
                               javax.xml.bind.annotation,\
                               javax.xml.bind.annotation.adapters,\
                               javax.lang.model,\
                               javax.lang.model.element,\
                               javax.xml.bind,\
-                              org.mule.runtime.module.extension.internal.runtime.operation
+                              reactor.core,\
```

Comment:
do we have an issue to track these packages that shouldn't be exported and clean them up at some point?